src/SourceBuild/content/NuGet.config

```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
+    The config is dynamically altered during the build to include sources such as locally built packages.
```

Comment:
I find the wording of `the config` to be a little misleading as to me it indicates this config is dynamically altered.  It is the individual repo nuget configs that are dynamically altered.

How about:

`This empty NuGet.config at the root of the repo ensures that a global/default one (referencing nuget.org) is not used by accident.  The individual repo NuGet.config files are utilized during the build.`